Description: Fix a spelling error
 Upstream code has a spelling error:
  - libavcodec/dcadec.c                 Inconsistant -> Inconsistent

Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Forwarded: <https://ffmpeg.org/pipermail/ffmpeg-devel/2014-March/155297.html>
Applied-Upstream: <https://ffmpeg.org/pipermail/ffmpeg-devel/2014-March/155304.html>
Last-Update: <2014-03-24>


--- ffmpeg-2.2.orig/libavcodec/dcadec.c
+++ ffmpeg-2.2/libavcodec/dcadec.c
@@ -2415,7 +2415,7 @@ FF_ENABLE_DEPRECATION_WARNINGS
          * masks in some sense -- unfortunately some channels could overlap */
         if (av_popcount(channel_mask) != av_popcount(channel_layout)) {
             av_log(avctx, AV_LOG_DEBUG,
-                   "DTS-XXCH: Inconsistant avcodec/dts channel layouts\n");
+                   "DTS-XXCH: Inconsistent avcodec/dts channel layouts\n");
             return AVERROR_INVALIDDATA;
         }
 
